src/entrypoints/entrypoints.ts

```diff
@@ -0,0 +1,142 @@
+import { AbiRegistry } from "../abi";
+import { AccountController } from "../accountManagement";
+import { IAccount } from "../accounts/interfaces";
+import { Address } from "../address";
+import { DelegationController } from "../delegation";
+import { ErrInvalidNetworkProviderKind } from "../errors";
+import { Message, MessageComputer } from "../message";
+import { ApiNetworkProvider, ProxyNetworkProvider, TransactionOnNetwork } from "../networkProviders";
+import { RelayedController } from "../relayed/relayedController";
+import { SmartContractController } from "../smartContracts/smartContractController";
+import { TokenManagementController } from "../tokenManagement";
+import { Transaction } from "../transaction";
+import { TransactionComputer } from "../transactionComputer";
+import { TransactionWatcher } from "../transactionWatcher";
+import { TransfersController } from "../transfers/transfersControllers";
+import { UserVerifier } from "../wallet";
+import { DevnetEntrypointConfig, MainnetEntrypointConfig, TestnetEntrypointConfig } from "./config";
+
+class NetworkEntrypoint {
+    private networkProvider: ApiNetworkProvider | ProxyNetworkProvider;
+    private chainId: string;
+
+    constructor(networkProviderUrl: string, networkProviderKind: string, chainId: string) {
+        if (networkProviderKind === "proxy") {
+            this.networkProvider = new ProxyNetworkProvider(networkProviderUrl);
+        } else if (networkProviderKind === "api") {
+            this.networkProvider = new ApiNetworkProvider(networkProviderUrl);
+        } else {
+            throw new ErrInvalidNetworkProviderKind();
+        }
+
+        this.chainId = chainId;
+    }
+
+    async signTransaction(transaction: Transaction, account: IAccount): Promise<void> {
+        const txComputer = new TransactionComputer();
+        transaction.signature = await account.sign(txComputer.computeBytesForSigning(transaction));
+    }
+
+    verifyTransactionSignature(transaction: Transaction): boolean {
+        const verifier = UserVerifier.fromAddress(Address.fromBech32(transaction.sender));
+        const txComputer = new TransactionComputer();
+        return verifier.verify(txComputer.computeBytesForVerifying(transaction), transaction.signature);
+    }
+
+    async signMessage(message: Message, account: IAccount): Promise<void> {
```

Comment:
I think they will be soon moved to account: https://github.com/multiversx/mx-sdk-specs/pull/94